src/test/php/util/data/unittest/SequenceTest.class.php

```diff
@@ -21,90 +21,91 @@ protected function assertNotTwice($seq, $func) {
     });
   }
 
-  #[@test]
+  #[Test]
   public function empty_sequence() {
     $this->assertSequence([], Sequence::$EMPTY);
   }
 
-  #[@test]
+  #[Test]
   public function toArray_for_empty_sequence() {
     Assert::equals([], Sequence::$EMPTY->toArray());
   }
 
-  #[@test, @values('util.data.unittest.Enumerables::validArrays')]
+  #[Test, Values('util.data.unittest.Enumerables::validArrays')]
   public function toArray_returns_elements_as_array($input, $name) {
     Assert::equals([1, 2, 3], Sequence::of($input)->toArray(), $name);
   }
 
-  #[@test]
+  #[Test]
   public function toArray_optionally_accepts_mapper() {
     Assert::equals(
       [2, 4],
       Sequence::of([1, 2])->toArray(function($v) { return $v * 2; })
     );
   }
 
-  #[@test]
+  #[Test]
   public function toMap_for_empty_sequence() {
     Assert::equals([], Sequence::$EMPTY->toMap());
   }
 
-  #[@test, @values('util.data.unittest.Enumerables::validMaps')]
+  #[Test, Values('util.data.unittest.Enumerables::validMaps')]
   public function toMap_returns_elements_as_map($input) {
     Assert::equals(['color' => 'green', 'price' => 12.99], Sequence::of($input)->toMap());
   }
 
-  #[@test]
+  #[Test]
   public function toMap_optionally_accepts_mapper() {
     Assert::equals(
       ['a' => 2, 'b' => 4],
       Sequence::of(['a' => 1, 'b' => 2])->toMap(function($v) { return $v * 2; })
     );
   }
 
-  #[@test, @values([
-  #  [0, []],
-  #  [1, [1]],
-  #  [4, [1, 2, 3, 4]]
-  #])]
+  #[Test, Values([[0, []], [1, [1]], [4, [1, 2, 3, 4]]])]
   public function count($length, $values) {
     Assert::equals($length, Sequence::of($values)->count());
   }
 
-  #[@test]
+  #[Test]
   public function first_returns_non_present_optional_for_empty_input() {
     Assert::false(Sequence::of([])->first()->present());
   }
 
-  #[@test]
+  #[Test]
   public function first_returns_present_optional_even_for_null() {
     Assert::true(Sequence::of([null])->first()->present());
   }
 
-  #[@test]
+  #[Test]
   public function first_returns_first_array_element() {
     Assert::equals(1, Sequence::of([1, 2, 3])->first()->get());
   }
 
-  #[@test]
+  #[Test]
   public function first_returns_first_element_to_match_its_filter() {
     Assert::equals(2, Sequence::of([1, 2, 3])->first(function($i) { return 0 === $i % 2; })->get());
   }
 
-  #[@test]
+  #[Test]
   public function first_returns_non_present_optional_if_no_element_matches_its_filter() {
     Assert::false(Sequence::of([1, 3])->first(function($i) { return 0 === $i % 2; })->present());
   }
 
-  #[@test, @values([
-  #  [[1, 2, 3], [1, 2, 2, 3, 1, 3]],
-  #  [[new Name('a'), new Name('b')], [new Name('a'), new Name('a'), new Name('b')]]
-  #])]
-  public function distinct($result, $input) {
```

Comment:
This was manually rewritten - the `new ...` expressions were refactored into a separate tests.